runtime/oti/j9nonbuilder.h

```diff
@@ -150,6 +150,7 @@
 #define J9StaticFieldRefFlagBits 0x1F
 #define J9StaticFieldRefIsolated 0x8
 #define J9StaticFieldRefPutResolved 0x10
+#define J9StaticFieldRefBoolean 0x20
```

Comment:
This is outside of the range of J9StaticFieldRefFlagBits. Suggest re-using the isolated value as that is no longer used.